<launch>
 
  <node pkg="rail_agile_nodes" type="ClickedImageClient" name="ClickedImageClient" output="screen"/>
    
  <node pkg="rail_agile_nodes" type="point_cloud_clicker" name="point_cloud_clicker" output="screen"/>
</launch>
